setup.cfg

```diff
@@ -25,8 +25,10 @@ package_dir=
     =src
 packages=find_namespace:
 install_requires =
+    deephaven
     deephaven-plugin
     matplotlib
+    jpy
```

Comment:
This is incorrect - it requires "deephaven-jpy", at least as we have it named now (what you see when you do a pip list on a DH install).